### PR TITLE
feat(review-apps): Add formation to Documentation

### PR DESCRIPTION
**Pull request by baptmaury**
Add documentation based on https://scalingo.com/blog/scalingo-json-formation

ticket: https://github.com/Scalingo/documentation/issues/2604

### CODE REVIEW

File: src/_posts/platform/app/2000-01-01-app-manifest.md [hunk 1]
```diff
@@ -110,6 +110,39 @@ The JSON manifest also lets you declare scripts which will be executed as [postd
 }
 ```
 
+## Container Formation
+
+The `scalingo.json` manifest can be used to setup a container formation. A container formation defines any type of container that will be launched on startup.
+
```
Comment (mcimbaro) [inline src/_posts/platform/app/2000-01-01-app-manifest.md:116]:
```suggestion
The `scalingo.json` manifest can be used to setup a container formation. A container formation defines the shape of the containers that will be launched.

```

`type` seems too vague, shape or size is more adequate IMO.
Furthermore, the word `on startup` doesn't seems to add anything, isn't it?

File: src/_posts/platform/app/2000-01-01-app-manifest.md [hunk 2]
```diff
@@ -110,6 +110,39 @@ The JSON manifest also lets you declare scripts which will be executed as [postd
...
+### Review Apps
+
+You can specify the number of containers of each type to be launched when a Review App is created. This is useful for making an exact copy of a production environment in Review Apps without having to perform any manual actions.
```
Comment (mcimbaro) [inline src/_posts/platform/app/2000-01-01-app-manifest.md:119]:
```suggestion
You can specify the shape and the number of containers to be launched when a Review App is created. This is useful for making an exact copy of a production environment in Review Apps without having to perform any manual actions.
```

File: src/_posts/platform/app/2000-01-01-app-manifest.md (hunk 1 above)
Comment (baptmaury) [reply src/_posts/platform/app/2000-01-01-app-manifest.md:116]: Your true :), I commit the suggestion

**Review by mcimbaro — APPROVED** [review]
lgtm!

**Pull request merged**